### ISSUE TITLE
Tests: Error: Class 'NullPointerException' not found

### ISSUE THREAD

**Issue by tenzap** [giggsey/libphonenumber-for-php#514]
While running a test, I get this error

```
1) libphonenumber\Tests\core\PhoneNumberMatchTest::testIllegalArguments
Error: Class 'NullPointerException' not found

php-giggsey-libphonenumber/src/PhoneNumberMatch.php:40
php-giggsey-libphonenumber/tests/core/PhoneNumberMatchTest.php:37

```

I'm no PHP expert, but does that `NullPointerException` exist in PHP? It is not highlighted in my editor and I can't find such a class in your code. It isn't in PHP doc either? https://www.php.net/manual/en/spl.exceptions.php & https://www.php.net/manual/en/reserved.exceptions.php

```
grep -r NullPointerException
tests/core/PhoneNumberMatchTest.php:        } catch (\NullPointerException $e) {
src/PhoneNumberMatcher.php:     * @throws \NullPointerException
src/PhoneNumberMatch.php:     * @throws \NullPointerException
src/PhoneNumberMatch.php:            throw new \NullPointerException;

```

**Comment by giggsey** [comment]
Good spot. My PHPStorm suggests it came from phing.

I'd be happy to receive a MR to replace those with something within SPL (possibly InvalidArgumentException)

**Comment by giggsey** [comment]
No worries. I'll leave the tags on there, and try to get to it within a week or so.

**Comment by giggsey** [comment]
Hi,

This bug fix is now part of a release (8.12.51)
